test/integration/google_sso_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+require 'omniauth_helper'
+
+class GoogleSSOTest < ActionDispatch::IntegrationTest
+  describe 'access top page' do
+    before do
+      mock_omniauth
+    end
+
+    it 'can sign in with Google Auth Account' do
+      @user = create :user, email: 'test@gmail.com'
```

Comment:
yeah! I think the mocking is pretty much completely agnostic to whether the google account actually exists or not which is nice.